channelmirror/channelmirror.py

```diff
@@ -60,8 +60,14 @@ async def add(self, ctx, source_channel_id: int, dest_channel_id: int, docheck:
         if docheck and (not self.bot.get_channel(source_channel_id) or not self.bot.get_channel(dest_channel_id)):
             await ctx.send(inline('Check your channel IDs, or maybe the bot is not in those servers'))
             return
+        conf = await confirm_message(ctx, "Set up mirror of <#{}> to mirror to <#{}>?"
+                                     .format(source_channel_id, dest_channel_id))
+        if not conf:
+            await ctx.send(":x: Action cancelled. No action was taken.")
```

Comment:
ok we now have send_cancellation_message and send_confirmation_message in tsutils!! update to 4.0.1 (and restart hathor) and we can use these instead